wherehows-web/app/components/browser/browser-viewport.ts

```diff
@@ -1,2 +1,17 @@
 import Component from '@ember/component';
-export default class extends Component {}
+import { action } from 'ember-decorators/object';
```

Comment:
yay!

Reply:
haha, `action` is the only decorator I trust at the moment @igbopie